contributed_definitions/nyaml/NXmpes_liquid.yaml

```diff
@@ -301,21 +124,21 @@ NXmpes_liquid(NXmpes):
             doc: "Is '.', since beam axis is assumed to be horizontal (perpendicular to gravity)."
           \@transformation_type(NX_CHAR):
             enumeration: ["rotation"]
-          \@units: NX_ANGLE
+          unit: NX_ANGLE
           \@vector(NX_NUMBER):
         rot_y(NX_NUMBER):
           \@depends_on(NX_CHAR):
             doc: "Link to /NXentry/NXsample/rot_x"
           \@transformation_type(NX_CHAR):
             enumeration: ["rotation"]
-          \@units: NX_ANGLE
+          unit: NX_ANGLE
           \@vector(NX_NUMBER):
         rot_z(NX_NUMBER):
           \@depends_on(NX_CHAR):
             doc: "Link to /NXentry/NXsample/rot_y."
           \@transformation_type(NX_CHAR):
             enumeration: ["rotation"]
-          \@units: NX_ANGLE
+          unit: NX_ANGLE
           \@vector(NX_NUMBER):
         leaf_normal(NXtransformations):
```

Comment:
Should this really be nested transformations or rather live besides the other transformations? I.e., sample/transformations/.... + sample/leaf_normal ....

Reply:
I think it's fine to have it as `leaf_norma(NX_NUMBER)` here (as discussed via mail). I was just confused because it is `NXtransformations` here.